src/utils.js

```diff
@@ -165,3 +166,42 @@ export function ensureDefinedConfig(object, requester) {
     }
   });
 }
+
+/**
+ * This function generates and returns the API config url based on the following environment variables:
+ *
+ *  MFE_CONFIG_API_PATH: API config path, this must not include the domain.
+ *  MFE_CONFIG_API_PORT(Optional): Specify the port, useful in development environments
+ *  MFE_CONFIG_API_REGEX: This regex will be used to generate the config domain based on the current domain.
+ *  APP_ID(Optional): This value makes reference to specific settings for a MFE.
+ *
+ * @returns {string}
+ */
+export function generateAPIConfigUrl() {
```

Comment:
would be this function used in other cases or only in MFE runtime config API?

Part of the feedback from the community members was: 

> If we add something to the utilities here, it should be something we expect MFEs will need; these are intended to be common operations that MFEs will generally all do. Since the function is just used internally, I think I'd almost rather we just inline it where we need it.